File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-0.cpp
class Solution {
   public:
    int findMin(vector<int>& nums) {
        int n = nums.size();
        int l = 0, r = n - 1;
        int minn = nums[0];
        int m = 0;
        while (l <= r) {
            m = l + (r - l) / 2;
            if (nums[l] < nums[r]) {
                return min(minn, nums[l]);
            }
            minn = min(minn, nums[m]);
            if (nums[l] <= nums[m])
             { l = m + 1; }
            else {
                r = m - 1;
            }
        }
        return minn;
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxx = 0
        temp =0
        for i in range(len(nums)):
            if nums[i] == 1:
                temp+=1
            else:
                maxx = max(maxx, temp)
                temp = 0
        maxx = max(maxx,temp)
        return maxx